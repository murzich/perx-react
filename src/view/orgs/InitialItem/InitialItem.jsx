import React from 'react';
import { List, Placeholder } from 'semantic-ui-react';

const InitialItem = () => (
  <List.Item style={{ display: 'flex', alignItems: 'flex-start' }}>
    <Placeholder style={{ height: 32, width: 32, marginRight: '1rem' }}>
      <Placeholder.Image />
    </Placeholder>
    <List.Content>
      <List.Header>
        Enter user name to search for organizations
      </List.Header>
      <List.Description>
        There should be a list of all his organizations with public access.
      </List.Description>
    </List.Content>
  </List.Item>
);

export default InitialItem;
