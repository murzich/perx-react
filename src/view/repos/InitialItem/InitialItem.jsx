import React from 'react';
import { List } from 'semantic-ui-react';

const InitialItem = () => (
  <List.Item>
    <List.Content>
      <List.Header>
        Enter user name to search for his repositories
      </List.Header>
      <List.Description>
        There should be a list of all his repositories with public access.
      </List.Description>
    </List.Content>
  </List.Item>
);

export default InitialItem;
