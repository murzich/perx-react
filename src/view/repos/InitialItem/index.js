import InitialItem from './InitialItem';

export default InitialItem;
