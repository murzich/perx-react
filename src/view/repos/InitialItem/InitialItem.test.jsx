import { cleanup, render } from '@testing-library/react';
import React from 'react';
import InitialItem from './InitialItem';

describe('InitialItem', () => {
  afterEach(cleanup);

  it('should match snapshot', () => {
    const { asFragment } = render(<InitialItem />);

    expect(asFragment()).toMatchSnapshot();
  });
});
