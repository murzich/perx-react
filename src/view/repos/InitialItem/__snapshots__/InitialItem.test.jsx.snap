// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InitialItem should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="item"
    role="listitem"
  >
    <div
      class="content"
    >
      <div
        class="header"
      >
        Enter user name to search for his repositories
      </div>
      <div
        class="description"
      >
        There should be a list of all his repositories with public access.
      </div>
    </div>
  </div>
</DocumentFragment>
`;
